utils/tex_bot.py

```diff
@@ -103,6 +105,27 @@ async def committee_role(self) -> discord.Role:
 
         return self._committee_role
 
+    @property
+    async def committee_elect_role(self) -> discord.Role:
+        """
+        Shortcut accessor to the "Committee-Elect" role.
+
+        The "Committee-Elect" role is the role held by committee members
+        after they have been elected, but before the handover period has concluded.
+
+        Raises `CommitteeElectRoleDoesNotExist` if the role does not exist.
+        """
+        if not self._committee_elect_role or not self._guild_has_role(self._committee_elect_role):  # noqa: E501
```

Comment:
Expand `if` statement so that line-length error does not need to be ignored.

Fix:
```python
COMMITTEE_ELECT_ROLE_NEEDS_FETCHING: Final[bool] = bool(
    not self._committee_elect_role
    or not self._guild_has_role(self._committee_elect_role)
)
if COMMITTEE_ELECT_ROLE_NEEDS_FETCHING:
    ...
```